ConsoleApplicationGoBoard/Console.cpp

```diff
@@ -38,57 +38,57 @@ const void Console::OutputDeadStones(GameState &goBoard)
 
 void Console::ObtainValidPlayerMove(GameState &goBoard)
 {
-	while (!GetCoordinatesAndPlay(goBoard))
-	{
-		std::cout << "you cant go there!" << std::endl;
-	}
+   while (!GetCoordinatesAndPlay(goBoard))
+   {
+      std::cout << "you cant go there!" << std::endl;
+   }
 }
 
 bool Console::GetCoordinatesAndPlay(GameState &goBoard)
 {
-	uint8_t size = goBoard.GoBoardSize();
-	int8_t x = size;
-	int8_t y = size;
-	while (x >= 0 && x >= size)
-	{
-		x = GetInput("please input an x coordinate : ");
-	}
-	while (y >= 0 && y >= size)
-	{
-		y = GetInput("please input an y coordinate : ");
-	}
+   uint8_t size = goBoard.GoBoardSize();
+   int8_t x = size;
+   int8_t y = size;
+   while (x >= 0 && x >= size)
+   {
+      x = GetInput("please input an x coordinate : ");
+   }
+   while (y >= 0 && y >= size)
+   {
+      y = GetInput("please input an y coordinate : ");
+   }
 
-	return goBoard.PlayStone(x, y, goBoard.WhosMoveIsNext());
+   return goBoard.PlayStone(x, y, goBoard.WhosMoveIsNext());
 }
 
 uint8_t Console::GetInput(string message)
 {
-	cout << message << endl;
-	string stringInput;
-	size_t* pos = 0;
-	uint8_t input = 0; 
-	uint8_t base = 10;
-	bool succesfull = false;
-	while (!succesfull)
-	{
-		cin >> stringInput;
-		try
-		{
-			input = stoi(stringInput, pos, base);
-			succesfull = true;
-		}
-		catch (invalid_argument)
-		{
-			cout << "Invalid!" << endl;
-			cout << message << endl;
-			succesfull = false;
-		}
-		catch (out_of_range)
-		{
-			cout << "Out of range!" << endl;
-			cout << message << endl;
-			succesfull = false;
-		}
-	}
-	return input;
+   cout << message << endl;
+   string stringInput;
+   size_t* pos = 0;
+   uint8_t input = 0;
+   uint8_t base = 10;
+   bool succesfull = false;
+   while (!succesfull)
+   {
+      cin >> stringInput;
+      try
+      {
+         input = stoi(stringInput, pos, base);
+         succesfull = true;
+      }
+      catch (invalid_argument)
+      {
+         cout << "Invalid!" << endl;
+         cout << message << endl;
+         succesfull = false;
+      }
+      catch (out_of_range)
+      {
+         cout << "Out of range!" << endl;
+         cout << message << endl;
+         succesfull = false;
+      }
+   }
+   return input;
 }
```

Comment:
untill successful = true then it dosent leave this code :) im confused at what you are trying to say :P